Misc/stable_abi.txt

```diff
@@ -2142,3 +2142,9 @@ const Py_am_send
 # (Detailed comments aren't really needed for further entries: from here on
 #  we can use version control logs.)
 
+function PyGC_Disable
+    added 3.10
+function PyGC_Enable
+    added 3.10
+function PyGC_IsEnabled
+    added 3.10
```

Comment:
Since these don't come as part of the original PR commit, I'd add a comment for them.

```suggestion
# New GC control functions in Py3.10 (https://bugs.python.org/issue28254)

function PyGC_Disable
    added 3.10
function PyGC_Enable
    added 3.10
function PyGC_IsEnabled
    added 3.10


# (Detailed comments aren't really needed for further entries: from here on
#  we can use version control logs.)
```

Reply:
Hmm, ah I see. you've linked them to the original PR. Yeah, that's enough then.

Reply:
When this PR is merged, it'll contain the bpo number, which is  IMO enough. But, nothing bad with being explicit.